src/rgw/rgw_http_client.cc

```diff
@@ -238,6 +238,37 @@ void RGWHTTPClient::_set_read_paused(bool pause)
   }
 }
 
+/*
+ * make attrs Look-Like-This
+ * converts underscores to dashes
+ */
+static string camelcase_dash_http_attr(const string& orig)
```

Comment:
@Leeshine this is just a copy paste of the same function in rgw_rest.cc. You can just put the prototype here if including `rgw_rest.h` is a problem.

Reply:
maybe we should move the related functions `camelcase_dash_http_attr,   lowercase_dash_http_attr,lowercase_underscore_http_attr, uppercase_underscore_http_attr` from rgw_rest to rgw_common

Reply:
sure